src/plone/app/multilingual/browser/interfaces.py

```diff
@@ -21,6 +21,11 @@ def make_relation_root_path(context):
     security_manager = getSecurityManager()
     site = getSite()
 
+    # Use the old behavior if context is None or REQUEST is missing, otherwise
+    # the code below will fail because REQUEST or aq_parent are missing
+    if context is None or not getattr(context, "REQUEST"):
```

Comment:
`getattr(context, "REQUEST")` will raise an AttributeError if there is no REQUEST attribute. You probably want `getattr(context, "REQUEST", None)`

Reply:
Ops, correct with https://github.com/plone/plone.app.multilingual/commit/75da825ee09331de6ce30e3820c960be24f72394